.github/workflows/ci.yml

```diff
@@ -125,9 +125,11 @@ jobs:
           eval `ssh-agent`
           RAILS_ENV=test bundle exec rspec --exclude-pattern spec/{features}/**/*_spec.rb spec
 
+      - name: Setup LCOV
```

Comment:
Necessary since version 3 due to https://github.com/zgosalvez/github-actions-report-lcov/pull/45/files#diff-27653c212e1cfe533e4eb2f7d0d3f89604c9de48a09583b4cbbbcbd08a07da79.

Reply:
updated PR description